client/src/components/modals/DetailsModal.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
You guys did an impeccable job with this modal component. This is a quality implementation right here 🔥